pyproject.toml

```diff
@@ -57,4 +57,4 @@ netam = "netam.cli:main"
 packages = ["netam"]
 
 [tool.setuptools.package-data]
-netam = ["py.typed"]
+netam = ["py.typed", "model_configs/*.yaml"]
```

Comment:
ah, here it is, package data! I think that's the thing I was thinking about.

Reply:
No, this is finished unless I’m missing something. Were you requesting a change?